src/VisualStudio/Core/Def/Telemetry/VisualStudioWorkspaceTelemetryService.cs

```diff
@@ -51,12 +52,15 @@ protected override void TelemetrySessionInitialized()
                     return;
                 }
 
-                var settings = SerializeCurrentSessionSettings();
-                Contract.ThrowIfNull(settings);
+                Contract.ThrowIfNull(CurrentSession);
+
+                var settings = CurrentSession.SerializeSettings();
+                var sharedProperties = ImmutableDictionary.CreateBuilder<string, string>();
+                sharedProperties.Add("VS.Core.Version", (string)CurrentSession.GetSharedPropertyAsObject("VS.Core.ExeVersion"));
```

Comment:
This version looks like "17.1.32104.313" on my machine. @AbhitejJohn is this the one we want?

Reply:
From education standpoint, why is VS.Core.ExeVersion set but not VS.Core.Version?

Reply:
yup, that format is what we have in our other tables as well.

Reply:
When the fault event is queried in Kusto, I believe this is the property we need to be set: context.default.vs.core.version.

Reply:
On the ExeVersion vs. Version, that's a perfectly good question and still trying to confirm that's correct on the internal email thread.